include/SDL_endian.h

```diff
@@ -109,7 +109,7 @@ extern "C" {
 
 #if HAS_BUILTIN_BSWAP16
 #define SDL_Swap16(x) __builtin_bswap16(x)
-#elif defined(_MSC_VER) && (_MSC_VER >= 1400)
+#elif defined(_MSC_VER) && (_MSC_VER >= 1400) && !defined(__clang__)
```

Comment:
clang-cl does not support _byteswap_uXXXX so these preprocessor checks were modified.

Reply:
Isn't `clang` case already handled by the previous `HAS_BUILTIN_BSWAPxx`?

Reply:
You are correct, it is already handled. However, I'm uncertain if __has_builtin could fail - maybe on a custom build of clang? If you want to consider that a fringe case, I'll go ahead and remove these checks.

Reply:
It _should_ be a fringe case if that _really_ happens (does it happen in any of your setups?)

Reply:
No, it doesn't occur. I'll go ahead and cut it.

Reply:
OK.